python/ray/_private/services.py

```diff
@@ -1460,12 +1460,12 @@ def start_gcs_server(
 
     Args:
         redis_address: The address that the Redis server is listening on.
-        log_dir: The path of the dir where log files are created.
-        session_name: The session name (cluster id) of this cluster.
-        stdout_file: A file handle opened for writing to redirect stdout to. If
-            no redirection should happen, then this should be None.
+        log_dir: The path of the dir where gcs event log files are created.
```

Comment:
```suggestion
        log_dir: The path of the dir where gcs log files are created.
```

Reply:
Updated, just one concern, `ray_log_filepath` variable name and the comment `gcs log files` is somehow similar.